docs/about.md

```diff
@@ -1,43 +0,0 @@
----
```

Comment:
The file wasn't deleted, it was just moved to the index. Since the information fits nicely in the index, it is superfluous to have an entire separate section for just a single paragraph.